Gemfile

```diff
@@ -12,6 +12,10 @@ branch = File.read(File.expand_path("../maintenance-branch", __FILE__)).chomp
   end
 end
 
+gem 'rspec-core', # https://github.com/rspec/rspec-expectations/pull/1080#issuecomment-429523851
+    :git => 'https://github.com/johnsyweb/rspec-core.git',
+    :branch => 'paj/replace-should-with-is-expected-to'
```

Comment:
@JonRowe I think I'm missing something in the pinning process here judging by https://travis-ci.org/rspec/rspec-expectations/jobs/441552090#L1216

Is this process documented anywhere for newbies like me?

Reply:
Honestly what you've done should have worked, I'm as puzzled as you

Reply:
I thought https://github.com/rspec/rspec-expectations/pull/1080/commits/3ef848f737e61179e52360b783c596c9df179fce might fix it but no. I'll have to come back to this some other time.

Reply:
Look like that failure was a Travis Glitch. https://travis-ci.org/rspec/rspec-expectations/builds/441874864 is green.

Reply:
OK... After 6 days I have a green build (but I'm not altogether convinced I took the correct path).

Reply:
> OK... After 6 days I have a green build (but I'm not altogether convinced I took the correct path).

Could you explain why? 😊

Reply:
@benoittgt : I thought 3ef848f would be enough to pass the CI checks. Editing the generated file at 0c0771d to get a successful build felt like a hack and reduced my confidence that my changes were correct. Perhaps that's just inexperience with rspec-dev.